docs/Tutorials/Quick-Start.md

```diff
@@ -168,4 +175,13 @@ echo $this->Form->file('file');
 echo $this->Form->error('file');
 echo $this->Form->submit(__('Upload'));
 echo $this->Form->end();
+```
+
+Products Image View
+-------------
+
+```php
+// found in src/Template/Products/view.php
+
+<?=  $this->Image->display($product->productImages[0],'medium',['width'=>'200px','height'=>'200px']); ?> 
```

Comment:
This will just render *one* image but the association is *hasMany*. Also it will cause a notice if there is no image present yet.

And this is **not** the view.ctp but **upload.ctp**.